views/sign_up.erb

```diff
@@ -0,0 +1,9 @@
+<form action='/sign_up' method='post'>
```

Comment:
I haven't used much <div> yet, but I was wondering if there is no need to create a Value for submit, to be able to click on the button?